index.js

```diff
@@ -228,28 +228,28 @@ console.log(artists[8])
  * it will return `The artist at index 0 is Amedeo Modigliani`.
 */
 function getArtistByIndex(array, index) {
-    return `The artist at index ${array[index].id} is ${array[index].name}`;
+  const names = `The artist at index ${array[index].id} is ${array[index].name}`;
+  return names
   }
-  getArtistByIndex(artists, 0);
+  console.log(getArtistByIndex(artists, 0));
   
-  console.log(artists, 0)
 
 
 /* Task 4: Create a function called get20s() that takes data as an argument and returns an array with names of artists who were born in and died in 20th century (1900-2000) example born in 1901 and died in 1959 - included / born in 1889 and died in 1925 not included - should return ["Salvador Dali", "Frida Kahlo"]*/
 
```

Comment:
{/*feedback
// youre on the right track here look into one of these array methods 
 - .includes('')
 - .substring()
 - .indexOf('') > -1

 I think that any of them could do what youre looking for
*/}